### PR TITLE
Feature loader fix execute

### PR DESCRIPTION
**Pull request by BazilMuralidharan**
URL for testing

- https://feature-loaderFixExecute--hdfc-bank--aemsites.hlx.page/content/forms/af/hdfc

### CODE REVIEW

**Review by kunal-adobe — APPROVED** [review]

File: common/executeinterfaceutils.js [hunk 1]
```diff
@@ -229,86 +244,119 @@ const resumeJourney = (globals, response) => {
...
   terminateJourney: (panStatus, globals) => {
     console.log(`pan Status: ${panStatus} and called terminateJourney()`);
-    terminateJourney(globals);
+    journeyTerminate(globals);
   },
 
-  restartJourney: (panStatus) => {
+  restartJourney: (panStatus, globals) => {
+    journeyRestart(globals);
     console.log(`pan Status: ${panStatus} and called restartJourney()`);
```
Comment (kunal-adobe) [inline common/executeinterfaceutils.js:360]: now console log is not required.

File: common/executeinterfaceutils.js [hunk 2]
```diff
@@ -229,86 +244,119 @@ const resumeJourney = (globals, response) => {
...
+            userAgent: navigator.userAgent,
+            journeyID: currentFormContext.journeyID,
+            journeyName: currentFormContext.journeyName,
+            productCode: currentFormContext.productCode,
+          },
+        };
+        TOTAL_TIME = 0;
+        sendIpaRequest(ipaRequestObj, globals);
+      } else {
+        console.log('terminate journey');
```
Comment (kunal-adobe) [inline common/executeinterfaceutils.js:337]: now console log is not required.

File: common/executeinterfaceutils.js (hunk 1 above)
Comment (BazilMuralidharan) [reply common/executeinterfaceutils.js]: consoles removed

File: common/executeinterfaceutils.js (hunk 2 above)
Comment (BazilMuralidharan) [reply common/executeinterfaceutils.js]: consoles removed

**Pull request merged**